postpic/io/image.py

```diff
@@ -70,40 +72,57 @@ def _readpng(filename):
     Args:
         filename (str): Path and filename to the file to open
 
-    kwargs:
-        hotpixelremove (bool): Removes hotpixels if True. (Default: True)
-
     Returns:
         numpy.array: the png data as numpy array converted to float64
 
-    Author: Stephan Kuschel, 2015-2016
+    Author: Stephan Kuschel, 2015-2016, Alexander Blinne 2019
     '''
-    import png  # pypng
-    import scipy.misc as sm
-    import numpy as np
+    have_pypng = False
+    data = _read_image(filename)
+    try:
+        import png  # pypng
+        have_pypng = True
+    except ImportError:
+        pass
+
+    if not have_pypng:
+        warnings.warn('Can not import `PyPNG`. Png file can be imported, however it is not '
+                      'guaranteed, that the count values will be correct for non 8 or 16 bit '
+                      'images.')
+        return data.astype(np.float64)
+
     meta = png.Reader(filename)
```

Comment:
How does this line work, in case the import failed?

Reply:
It won't get executed, because the function exits in the line above.